specification.markdown

```diff
@@ -40,17 +40,17 @@ This document defines specific terms in order to define the format and applicati
 
 `Element Name`: The official human-readable name of the `EBML Element`.
 
-`Empty Element`: An `Empty Element` is an `EBML Element` that has an `Element Data Size` with all VINT\_DATA bits set to zero which indicates that the `Element Data` of the Element is zero octets in length.
+`Element Path`: The hierarchy of `Parent Element` where the `EBML Element` is expected to be found in the `EBML Body`.
 
-`Level`: `Level` notes at what hierarchical depth the `EBML Element` may occur within an `EBML Document`, starting from zero.
+`Empty Element`: An `Empty Element` is an `EBML Element` that has an `Element Data Size` with all VINT\_DATA bits set to zero which indicates that the `Element Data` of the Element is zero octets in length.
 
 `Master Element`: The `Master Element` contains zero, one, or many other `EBML Elements`.
 
 `Parent Element`: A relative term to describe the `Master Element` which contains a specified element.
 
-`Root Element`: A mandatory, non-repeating `EBML Element` which occurs within an `EBML Document` at `Level 0` and contains all other `EBML Elements` of the `EBML Document`, excepting the `EBML Header` and optional `Void Elements`.
+`Root Element`: A mandatory, non-repeating `EBML Element` which occurs at the top level of the path hierarchy within an `EBML Body` and contains all other `EBML Elements` of the `EBML Body`, excepting optional `Void Elements`.
 
-`Top-Level Element`: An `EBML Element` defined to only occur at `Level 1` within the `Root Element`.
+`Top-Level Element`: An `EBML Element` defined as occuring within the `Root Element`.
```

Comment:
I don't think this is as clear as `Level 1` was. How about `defining as a Child Element of the Root Element`.